applications/clover_network.md

```diff
@@ -0,0 +1,182 @@
+Unit 8, 3/F, Qwomar Trading Complex, Blackburne Road, Port Purcell, Road Town, Tortola, British Virgin Islands, VG1110
+
+### Team's experience
+As a collective, the founding team has been working together for 8 months that has been in the industry for over 2 years, of which have crossed paths multiple times in different industry conferences. Within the team Burak is mainly based in the Bay Area, Chris in Hong Kong, Norelle in Canada, and Barek is based in the UK. 
+
+We all come from different and unique walks of life, backgrounds, and experiences which really makes us a dynamic and strong team complementing one another's skill sets. It is neat operating as a team across the globe because we can tap into different markets and bring different things to the table. Our team has been formed to optimize and complement one another’s skill sets; from DLT engineering to economics, to operational, and marketing expertise. We have a team capable of building, executing, and scaling Clover to its fullest potential. 
+
+### Team Code Repos
+* [https://github.com/clover-network/clover](https://github.com/clover-network/clover)
+
+* [https://github.com/clover-network/clover-web3-wallet](https://github.com/clover-network/clover-web3-wallet)
+
+* [https://github.com/clover-network/website](https://github.com/clover-network/website)
+
+* [https://github.com/clover-network/clover-defi-wallet](https://github.com/clover-network/clover-defi-wallet)
+
+* [https://github.com/clover-network/clover-sdk](https://github.com/clover-network/clover-sdk)
+
+* [https://github.com/clover-network/clover-wallet](https://github.com/clover-network/clover-wallet)
+
+* [https://github.com/clover-network/clover-mobile-wallet](https://github.com/clover-network/clover-mobile-wallet)
+
+* [https://github.com/clover-network/clover-dapp-uniswap](https://github.com/clover-network/clover-dapp-uniswap)
+
+* [https://github.com/clover-network/clover-store-core](https://github.com/clover-network/clover-store-core)
+
+* [https://github.com/clover-network/clover-store-sdk](https://github.com/clover-network/clover-store-sdk)
+
+* [ttps://github.com/clover-network/clover-store-upload](https://github.com/clover-network/clover-store-upload)
+
+### Team LinkedIn Profiles
+* [Burak Keceli](https://www.linkedin.com/in/burak-ke%C3%A7eli/)
+
+* [Chris Li](https://www.linkedin.com/in/chrislky/)
+
+* [Norelle Ng](https://www.linkedin.com/in/norelleng/)
+
+* [Chris Williamson](https://www.linkedin.com/in/chris-williamson-aa90b290/)
+
+* [Barek Sekandari](https://www.linkedin.com/in/barek-sekandari-597752153/)
+
+* [Marina Danylyuk](https://www.linkedin.com/in/marinadanylyuk/)
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 3 months
+* **Full-time equivalent (FTE):**  10 FTE
+* **Total Costs:** Amount of Payment in BTC for the whole project. The total amount of funding needs to be below $30k at the time of submission. (e.g. 0.80 BTC)
```

Comment:
Could you update this to show the complete total costs? (0.66 BTC)

Reply:
Just updated. Thanks.